README.md

```diff
@@ -29,3 +29,17 @@ Required environment variables:
   HODIS_HOST=https://hodis.datasektionen.se
   LOGIN_HOST=https://login2.datasektionen.se
 ```
+
+## Dependencies
+Depends on login, hodis is refered to but not integral.
```

Comment:
```suggestion
Depends on login, hodis is referred to but not integral.
```

Reply:
Hodis is very integral. That is where zfinger fetches all information about users. Without hodis, zfinger would be pointless.

EDIT: @DokID I will forever write referer, this is HTTP's fault.

Reply:
Well... technically, zfinger would work fine. Doesn't display anything, but it doesn't crash, and the app itself works fine.

Reply:
I don't agree. The point of zfinger is to look up people, that's the core functionality. Without hodis that functionality is impossible. So hodis is very much integral to zfinger.